lesson02/canDrive.js

```diff
@@ -0,0 +1,17 @@
+// bonus js: lesson 02 challenge 4 
+
+let firstName = 'John';
+let lastName = 'Smith';
+const age = 15;
+const driving_age = 16;
+
+// string interpolation
+msg = `${firstName} ${lastName} can drive, he/she has been driving for ${age - driving_age} years`;
```

Comment:
Your msg variable has not been declared before, so you must preface msg with either let or const. `let msg = ""`